src/routers/statisticRouter.js

```diff
@@ -2,8 +2,8 @@ const { Router } = require('express');
 const statisticController = require('../controllers/statisticController');
 
 const statisticRouter = Router();
-statisticRouter.get('/ARR/:year', statisticController.getARR);
-statisticRouter.get('/MRR/:year/:month', statisticController.getMRR);
-statisticRouter.get('/DRR/:year/:month/:day', statisticController.getDRR);
+statisticRouter.get('/arr/:year', statisticController.getARR);
+statisticRouter.get('/mrr/:year/:month', statisticController.getMRR);
+statisticRouter.get('/drr/:year/:month/:day', statisticController.getDRR);
```

Comment:
As far as I understand from David's notes we should change it and supply only one route of these statistics